org.eclipse.lsp4j/src/test/java/org/eclipse/lsp4j/test/util/PositionsTest.java

```diff
@@ -0,0 +1,62 @@
+/******************************************************************************
+ * Copyright (c) 2018 TypeFox and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ ******************************************************************************/
+package org.eclipse.lsp4j.test.util;
+
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.util.Positions;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PositionsTest {
+
+	@Test(expected = NullPointerException.class)
+	public void isBefore_nullLeft() {
+		Positions.isBefore(null, new Position(0, 0));
+	}
+
+	@Test(expected = NullPointerException.class)
+	public void isBefore_nullRight() {
+		Positions.isBefore(new Position(0, 0), null);
+	}
+
+	@Test
+	public void isBefore_equals() {
+		Assert.assertFalse(Positions.isBefore(new Position(3, 3), new Position(3, 3)));
```

Comment:
This is incorrect.

Reply:
Why? It looks good to me?

Reply:
It is checking "same" ranges, not "equals". Technically, they're the same though.